src/components/Landing.jsx

```diff
@@ -0,0 +1,7 @@
+import React, { Component } from "react";
+
+export default class Landing extends Component {
+  render() {
+    return <div>Landing</div>;
```

Comment:
what is the use of this component when you only have a single tag with just text??